source/view/index.js

```diff
@@ -1567,4 +1567,10 @@ function writeCameraPose( radius, yaw, pitch ) {
 
 window.addEventListener( "resize", checkPageZoom );
 
+$( document ).keydown( function( objEvent ) {
+    if ( objEvent.keyCode == 9 ) {  //tab pressed
+        objEvent.preventDefault(); // stops its action
+    }
+})
```

Comment:
Nitpick: Semicolon missing at the end and a space between the function bracket and parenthesis.

It's interesting that this seems to work. I tested the same thing without jQuery and it had no effect. The other possible difference is the context. I used `window` key events as opposed to `document`.